ampd/src/handlers/solana_verify_msg.rs

```diff
@@ -191,3 +205,296 @@ where
         self.broadcast_votes(poll_id, votes).await
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
So this is tests are not very extensive. Thats because the verification logic is tested [here](https://github.com/ctoyan/axelar-amplifier-fork/pull/3/files#diff-58440ccf431024a6540687ee4e0982f044583bb4eebd18e76be09a26989c7ce5R235), where unit tests have much more definition. Here we only check we are calling the proper subcomponents.